### ISSUE TITLE
Update lib for kotlin1.4.30

### ISSUE THREAD

**Issue by dah0f** [MicroUtils/kotlin-logging#168]
First of all, thanks four the great lib. Can you provide the lib also for kotlin 1.4.30? Compiling fails after updating the kotlin multiplatform version from 1.4.10 to 1.4.30.

**Comment by oshai** [comment]
what is the error?

**Comment by dah0f** [comment]
i get a error on package import: "unresolved reference: mu"


part of my gradle:
```
plugins {
    kotlin("multiplatform") version "1.4.30"
   //...
}

kotlin {
    jvm {
        compilations.all {
            kotlinOptions.jvmTarget = "1.8"
        }
        testRuns["test"].executionTask.configure {
            useJUnit()
        }
    }
    sourceSets {
        val commonMain by getting {
            dependencies {
                implementation("io.github.microutils:kotlin-logging:2.0.3")
                implementation("org.jetbrains.kotlinx:kotlinx-serialization-json:1.0.0")
                implementation("org.jetbrains.kotlinx:kotlinx-datetime:0.1.1")
            }
        }
}
```

**Comment by oshai** [comment]
Yes, I will release a new version.

**Comment by oshai** [comment]
released in version 2.0.5

**Comment by dah0f** [comment]
Thank you very much!
